specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/IpTagsUpdate.json

```diff
@@ -0,0 +1,51 @@
+{
+    "parameters": {
+      "ipTagName": "ipTags1",
+      "resourceGroupName": "rg1",
+      "api-version": "2019-07-01",
+      "subscriptionId": "subid",
+      "ipTagsParameters": {
+        "tags": {
+          "key1": "value1",
+          "key2": "value2"
+        }
+      }
+    },
+    "responses": {
+      "200": {
+        "body": {
+          "name": "iptag1",
+          "id": "/subscriptions/subId/providers/Microsoft.Network/resourceGroup/myResourceGroup/ipTags/iptag1",
```

Comment:
Resource group name in parameters was specified as `rg1` but here it's `myResourceGroup`. Same mismatch with `ipTagName` and the case of subscriptionId differs
This also applies to the other examples, need to sync values in request and response